### PR TITLE
Increase max deletion size to 5000 (fixes #8)

### LINKED ISSUE #8: Increase max deletion size of 1000
This value has been reached already. We should increase to ~5k.

### PR DESCRIPTION
**Pull request by BenWu**
What's the purpose of the limit?  Would it be better to paginate and not have a limit?

### CODE REVIEW

**Comment by fbertsch** [comment]
@Ben-Wu it would be better to paginate, then we can remove the limit entirely. You up for doing that instead?

**Comment by BenWu** [comment]
Yes I'll do that instead.

**Comment by BenWu** [comment]
Using pagination now.  Tests aren't really confidence inspiring but I can't think of a great way to do them.  A test run on airflow works though.

File: tests/test_leanplum_export.py [hunk 1]
```diff
@@ -93,7 +98,7 @@ def set_contents(filename):
 
         mock_bucket, mock_client, mock_blob = Mock(), Mock(), Mock()
 
-        mock_client.list_blobs.side_effect = [[]]
+        mock_client.list_blobs.return_value = BlobList([])
```
Comment (fbertsch) [inline tests/test_leanplum_export.py]: Alternatively, use `Mock()` and then set the `return_value` for `pages`

**Review by fbertsch — APPROVED** [review]
Thanks for testing a run, it's hard to have tests for these API calls. We need a library for GCP akin to `moto`.

File: tests/test_leanplum_export.py (hunk 1 above)
Comment (BenWu) [reply tests/test_leanplum_export.py]: Ah `PropertyMock` was what I was forgetting about.

**Pull request merged**